extensions/default/src/utils/validations/areAllImageComponentsEqual.ts

```diff
@@ -0,0 +1,23 @@
+import toNumber from '@ohif/core/src/utils/toNumber';
+
+/**
+ * Check if all voxels in series images has same number of components (samplesPerPixel)
+ * @param {*} instances
+ * @returns
+ */
+export default function areAllImageComponentsEqual(
+  instances: Array<any>
+): boolean {
+  const firstImage = instances[0];
```

Comment:
For this and all of the other validation functions, consider adding a safety check of something like 

```
if (!instances?.length) { 
  return false;
}
```